### PR TITLE
Release 160.0.0

### PR DESCRIPTION
**Pull request by adonesky1**
## Explanation

**Patch release**
- `@metamask/selected-network-controller` to v15.0.1

## Checklist

- [ ] I've updated the test suite for new or updated code as appropriate
- [ ] I've updated documentation (JSDoc, Markdown, etc.) for new or updated code as appropriate
- [ ] I've highlighted breaking changes using the "BREAKING" category above as appropriate

### CODE REVIEW

File: packages/selected-network-controller/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [15.0.1]
+
+### Fixed
+
+- No longer add domains that have been granted permissions to `domains` state (nor create for it a selected network proxy) unless the `useRequestQueuePreference` flag is true ([#4368](https://github.com/MetaMask/core/pull/4368))
```
Comment (adonesky1) [inline packages/selected-network-controller/CHANGELOG.md:14]:
```suggestion
- No longer add domains that have been granted permissions to `domains` state (nor create for them a selected network proxy) unless the `useRequestQueuePreference` flag is true ([#4368](https://github.com/MetaMask/core/pull/4368))
```

File: packages/signature-controller/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [19.0.0]
+
+### Removed
+
+- **BREAKING:** Methods related to `eth_sign` signatures - `unapprovedMsgCount` getter, `messages` getter, `newUnsignedMessage` - have been removed. ([#4319](https://github.com/MetaMask/core/pull/4319))
+- constructor argument `isEthSignEnabled` is no longer expected. ([#4319](https://github.com/MetaMask/core/pull/4319))
```
Comment (adonesky1) [inline packages/signature-controller/CHANGELOG.md:15]:
```suggestion
- Constructor argument `isEthSignEnabled` is no longer expected. ([#4319](https://github.com/MetaMask/core/pull/4319))
```

File: packages/signature-controller/CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [19.0.0]
+
+### Removed
+
+- **BREAKING:** Methods related to `eth_sign` signatures - `unapprovedMsgCount` getter, `messages` getter, `newUnsignedMessage` - have been removed. ([#4319](https://github.com/MetaMask/core/pull/4319))
```
Comment (mcmire) [inline packages/signature-controller/CHANGELOG.md:14]: Both of these changes were already released in 17.0.0.

File: packages/preferences-controller/CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [14.0.0]
+
+### Removed
+
+- **BREAKING:** `setDisabledRpcMethodPreference` method has been removed, and `disabledRpcMethodPreferences` property has been removed from state ([#4319](https://github.com/MetaMask/core/pull/4319))
```
Comment (mcmire) [inline packages/preferences-controller/CHANGELOG.md:14]: This was already released in 12.0.0.

**Review by mcmire — DISMISSED** [review]

File: packages/assets-controllers/CHANGELOG.md [hunk 5]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [33.0.0]
+
+### Changed
+
+- **BREAKING** Bump peer dependency on `@metamask/preferences-controller` to `^14.0.0` ([#4374](https://github.com/MetaMask/core/pull/4374))
+
```
Comment (jiexi) [inline packages/assets-controllers/CHANGELOG.md:15]: looks like a few other changes need to be documented here

https://github.com/MetaMask/core/compare/@metamask/assets-controllers@32.0.0...HEAD

File: packages/queued-request-controller/CHANGELOG.md [hunk 6]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.13.0]
+
+### Changed
+
+- **BREAKING:** Bump peer dependency `@metamask/selected-network-controller` to `^14.0.0` ([#4374](https://github.com/MetaMask/core/pull/4374))
```
Comment (mcmire) [inline packages/queued-request-controller/CHANGELOG.md:14]: This was also done in 0.12.0 — in fact the peer dependency is up to `^15.0.0` now.

It looks like we are bumping the `devDependency` in this PR to match the new release of `selected-network-controller`. I am not sure whether it should be a dev dependency, but that is what it is today. So perhaps we do not need a new release of `queued-request-controller`? Or if that seems wrong, perhaps we ought to move it from `devDependencies` to `dependencies` in another PR and then we can make a new release of `queued-request-controller` that makes use of the new version of `selected-network-controller`.

**Comment by mcmire** [comment]
Can we name this PR "Release 160.0.0"? The release workflow won't activate unless the PR title matches the patterns set here: https://github.com/MetaMask/core/blob/main/.github/workflows/main.yml#L38

File: packages/queued-request-controller/CHANGELOG.md (hunk 6 above)
Comment (adonesky1) [reply packages/queued-request-controller/CHANGELOG.md:14]: oh sure enough only types are imported here... So yeah maybe we don't need to release?

Comment (mcmire) [reply packages/queued-request-controller/CHANGELOG.md:14]: Yeah, you may not need to. You might try upgrading the extension to use the latest version of these packages and seeing what happens.

Comment (adonesky1) [reply packages/queued-request-controller/CHANGELOG.md]: OK yeah I don't think we need to release `QueuedRequestController`

File: packages/selected-network-controller/CHANGELOG.md (hunk 1 above)
Comment (jiexi) [inline packages/selected-network-controller/CHANGELOG.md:14]:
```suggestion
- No longer add domains that have been granted permissions to `domains` state (nor create a selected network proxy for it) unless the `useRequestQueuePreference` flag is true ([#4368](https://github.com/MetaMask/core/pull/4368))
```

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Review by jiexi — APPROVED** [review]

File: packages/selected-network-controller/CHANGELOG.md (hunk 1 above)
Comment (jiexi) [reply packages/selected-network-controller/CHANGELOG.md:14]: not important. feel free to ignore

Comment (adonesky1) [reply packages/selected-network-controller/CHANGELOG.md:14]: I think both ways work haha but we can go with yours

**Review by Gudahtt — APPROVED** [review]

**Pull request merged**